src/main/kotlin/com/github/pool_party/pull_party_bot/commands/handlers/callback/DeleteNodeSuggestionCallback.kt

```diff
@@ -0,0 +1,39 @@
+package com.github.pool_party.pull_party_bot.commands.handlers.callback
+
+import com.elbekD.bot.Bot
+import com.elbekD.bot.types.CallbackQuery
+import com.github.pool_party.pull_party_bot.commands.Callback
+import com.github.pool_party.pull_party_bot.commands.CallbackAction
+import com.github.pool_party.pull_party_bot.commands.messages.ON_PARTY_DELETE_SUCCESS
+import com.github.pool_party.pull_party_bot.commands.messages.ON_PERMISSION_DENY_CALLBACK
+import com.github.pool_party.pull_party_bot.commands.validateAdministrator
+import com.github.pool_party.pull_party_bot.database.dao.PartyDao
+
+class DeleteNodeSuggestionCallback(private val partyDao: PartyDao) : Callback {
```

Comment:
Feels like there is a lot of common code with `DeleteSuggestionCallback` class, you can reformat it if you have a will